pyrate/prepifg.py

```diff
@@ -78,22 +79,42 @@ def main(params):
     process_ifgs_paths = np.array_split(ifg_paths, mpiops.size)[mpiops.rank]
     do_prepifg(process_ifgs_paths, exts, params)
 
-    if params[pyrate.constants.LT_FILE] is not None:
+    if params[C.LT_FILE] is not None:
         log.info("Calculating and writing geometry files")
-        mpiops.run_once(_write_geometry_files, params, exts, transform, ifg_paths[0])
+        mpiops.run_once(__write_geometry_files, params, exts, transform, ifg_paths[0].sampled_path)
     else:
         log.info("Skipping geometry calculations: Lookup table not provided")
 
+    if params[C.COH_FILE_LIST] is not None:
+        log.info("Calculating and writing geometry files")
+        mpiops.run_once(__calc_coherence_stats, params, ifg_paths[0].sampled_path)
```

Comment:
Can this be extended to run with `tiles_split`? It'll be a big computation to hold in memory when we need to hold hundreds of coherence files in memory, each with 1M pixels.

Reply:
To be implemented in a separate PR